slapp/transforms/video_utils.py

```diff
@@ -59,10 +59,16 @@ def transform_to_mp4(video: np.ndarray, output_path: str,
     Returns:
 
     """
+
+    # ffmpeg expects the video shape in width, height not row, col
+    # have to reverse shape when inputting
+    # gray8 is uint8 format
+
     writer = mpg.write_frames(output_path,
-                              video[0].shape,
-                              pix_fmt_in="gray",
-                              pix_fmt_out="gray",
+                              (video[0].shape[1], video[0].shape[0]),
+                              pix_fmt_in="gray8",
```

Comment:
Are we sure that the input video frames are in uint8 format? Oftentimes (but not always) microscopy data is stored in uint16 format so by specifying uint8 you may be cutting off or overflowing signal for pixels.

- [ ] Double check what data type 2P frames are stored as in h5 files

Reply:
It's cutting off, the ones I looked at are all 16 bit, they aren't uint though. At least the array dtype is not uint, I can't see why they would be negative however.
I made the change to gray16, alternatively we could normalize all videos to uint8 or uint16.

Reply:
The transform pipeline should be normalizing the video into `uint8` here:
https://github.com/AllenInstitute/segmentation-labeling-app/blob/2c04c1e9d60975a555ba9ce7aa7aaeca2bfe3789/slapp/transforms/transform_pipeline.py#L163-L164

Reply:
Great, the switching to int16 bitrighting was causing failing tests, I'm going to change it back to gray8.

Reply:
I had forgotten about the normalization. This is fine with me then.